running-commands-remotely.md

```diff
@@ -0,0 +1,132 @@
+# Running Commands Remotely
+
+## Using an SSH connection
+
+WP-CLI accepts an `--ssh=<host>` global parameter for running a command against a remote WordPress install.
+
+Under the hood, WP-CLI proxies commands to the ssh executable, which then passes them to WP-CLI installed on the remote machine. Your syntax for `--ssh=<host>` can be any of the following:
```

Comment:
Giving the format spec `[<user>@]<host>[:<port>][<path>]` would be good. Mentioning it's similar to `scp` and `git`. Mentioning `port` in list. And "to the WP-CLI installed" perhaps.